### PR TITLE
pd1 info update, add info on how to use ipmi

### PR DESCRIPTION
**Pull request by akshayknarayan**

### CODE REVIEW

File: hosts/pd_cluster.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ The local ip address of each machine is 10.1.1.X (e.g., pd2 is 10.1.1.2).
 Additionally, there is a supplementary 1G ethernet connection for each machine.
 These interfaces are called "em2" on each machine, with IP addresses 192.168.1.X (e.g., pd2 is 192.168.1.2).
 
+### Using IPMI
+
+`sudo ipmi pd<i> power cycle`
```
Comment (jonhoo) [inline hosts/pd_cluster.md]: This should probably mention that you have to do it from `jerry`? And also `screen -x root/pd<i>`

Comment (akshayknarayan) [reply hosts/pd_cluster.md]: `jarvis` right?

File: hosts/pd_cluster.md [hunk 2]
```diff
@@ -35,6 +35,12 @@ The local ip address of each machine is 10.1.1.X (e.g., pd2 is 10.1.1.2).
 Additionally, there is a supplementary 1G ethernet connection for each machine.
 These interfaces are called "em2" on each machine, with IP addresses 192.168.1.X (e.g., pd2 is 192.168.1.2).
 
+### Using IPMI
+
+On `jarvis`:
```
Comment (jonhoo) [inline hosts/pd_cluster.md]: Maybe also link https://github.com/mit-pdos/jarvis

Comment (akshayknarayan) [reply hosts/pd_cluster.md]: sure.

**Pull request merged**